client/dashboard/sites/scan/actions.tsx

```diff
@@ -0,0 +1,21 @@
+import { __experimentalVStack as VStack } from '@wordpress/components';
+import { Action } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { ThreatDescription } from './components/threat-description';
+import { ThreatsDetailCard } from './components/threats-detail-card';
+import type { Threat, Site } from '@automattic/api-core';
+
+export function getViewDetailsAction( site: Site ): Action< Threat > {
+	return {
+		id: 'view_details',
+		label: __( 'View details' ),
+		modalHeader: __( 'View threat details' ),
+		supportsBulk: false,
+		RenderModal: ( { items } ) => (
+			<VStack spacing={ 4 }>
+				<ThreatsDetailCard threats={ items } />
+				<ThreatDescription threat={ items[ 0 ] } site={ site } />
+			</VStack>
+		),
+	};
+}
```

Comment:
Hi @Automattic/lego 👋, I would like some early feedback in here :)

We need to make the `view_details` action available in the two Scan dataviews (Active Threats and History). Just wanted to double-check if this is the best approach for it.

* scan
  * actions.tsx <- shared actions (only one right now)
* scan-active
  * dataviews/
    * actions.tsx <- active threats dataviews actions
* scan-history
  * dataviews/ 
    * actions.tsx <- history dataviews actions

Reply:
It looks fine, maybe also put `actions.tsx` in `scan/dataviews` for symmetry.